db/system_keyspace.cc

```diff
@@ -257,6 +257,7 @@ schema_ptr system_keyspace::topology() {
             .with_column("shard_count", int32_type)
             .with_column("ignore_msb", int32_type)
             .with_column("new_cdc_generation_data_uuid", uuid_type)
+            .with_column("version", long_type, column_kind::static_column)
```

Comment:
How does this survive upgrades?

Given this is a raft table, do we need to coordinate it with other nodes? 

Note that 5.3 will be branched soon, so we must be prepared for 5.3->master upgrades.

Reply:
The table is behind experimental. Should not be an issue.